### ISSUE TITLE
Selenium Automated Testing cannot catch same violation with aXe devTools plugin

### ISSUE THREAD

**Issue by Tim-Cao** [dequelabs/axe-core-maven-html#261]
**Description**
Manual Testing using [aXe devTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?utm_source=deque.com&utm_medium=referral&utm_campaign=axe-devtools_pricing_free) with **WCAG 2.1 AA** standard and the **latest** axe-core to scan [this page](https://www.liferay.com/) can catch some accessibility violations

However, when automated testing using the axe deque selenium api cannot get the above violations.
Would you mind explaining what's the difference between them?

**Step to reproduce**

1. Clone this repo [git@github.com:Tim-Cao/seleniumproject.git](https://github.com/Tim-Cao/seleniumproject)
2. Change the chromedriver path in https://github.com/Tim-Cao/seleniumproject/blob/master/src/test/java/com/example/seleniumproject/AccessibilityTesting.java
3. Run the above test

**Actual Results**
No violations thrown
![AutomatedTesting](https://user-images.githubusercontent.com/52661397/198509283-0cb0f92c-3590-4cad-b7bf-bfe61d6a7d4d.png)

**Expected Results**
![ManualTesting](https://user-images.githubusercontent.com/52661397/198509005-68390fd1-6a5f-47ee-869b-440713703a33.png)

**Comment by Zidious** [comment]
Hey @Tim-Cao, 

I've investigated this and found a few things, within your Gradle file you are using a older version of our API [V4.3.1](https://github.com/Tim-Cao/seleniumproject/blob/master/build.gradle#L30) which utilises axe-core 4.3.1 vs the extension using axe-core 4.4.2. 

I'd recommend to update to [v4.4.2 of our Selenium API ](https://search.maven.org/artifact/com.deque.html.axe-core/selenium/4.4.2/jar). 

I noticed within your [test](https://github.com/Tim-Cao/seleniumproject/blob/master/src/test/java/com/example/seleniumproject/AccessibilityTesting.java#L46): 

```java
withTags(Arrays.asList("wcag21aa"))
```

One important thing to note here, specifying `wcag21aa` _only_ runs rules associated to WCAG2.1aa. 

Selecting WCAG 2.1AA via the extension includes: WCAG2.0A/AA and WCAG2.1A. To mimic this behaviour with the selenium API you would need to include the following tags: 

```java
Arrays.asList("wcag2a", "wcag2aa", "wcag21a", "wcag21aa")
```

Running the below snippet produces four rule violations and one incomplete (needs review within the extension) summing the nodes of each violation totals 10 issues (breakdown below): 

Snippet:

```java
    WebDriver webDriver = new ChromeDriver();

    webDriver.get("https://www.liferay.com/");

    AxeBuilder axeBuilder = new AxeBuilder();
    axeBuilder.withTags(Arrays.asList("wcag2a", "wcag2aa", "wcag21a", "wcag21aa"));

    /* accept cookies popup */
    webDriver.findElement(By.cssSelector("#onetrust-accept-btn-handler")).click();

    Results results = axeBuilder.analyze(webDriver);

    List<Rule> violations = results.getViolations();

    /* denoted as needs review within the extension */
    List<Rule> incomplete = results.getIncomplete();

    int violationNodeCount = violations.stream().mapToInt(v -> v.getNodes().size()).sum();
    int incompleteNodeCount = incomplete.stream().mapToInt(v -> v.getNodes().size()).sum();

    System.out.println("Sum of violations (nodes): " + violationNodeCount);
    System.out.println("Sum of incomplete (nodes): " + incompleteNodeCount);
    System.out.println("Total: " + (violationNodeCount + incompleteNodeCount));

    AxeReporter.writeResultsToJsonFile("./liferay.json", results);
    webDriver.close();
```

Outputs: 

```sh
Sum of violations (nodes): 8
Sum of incomplete (nodes): 2
Total: 10
```

**Comment by Tim-Cao** [comment]
Many thanks @Zidious. It works now.
